l10n_nl_country_states/__scripts__/generate_postcode_regios.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
```

Comment:
Would it be possible to sort the excell sheel by zip code?

Then you could safe the province (res.country.state), min_zip encountered and max_zip encounterd until province changes. And only then write range. This would I think significantly reduce the resulting file in size, while still supporting having several ranges within one province (and a range laying between in another province).

Reply:
sure, that's the whole point of the data model. Just didn't come to this yet